test/HealthChecks.Redis.Tests/Functional/RedisHealthCheckTests.cs

```diff
@@ -5,12 +5,19 @@
 
 namespace HealthChecks.Redis.Tests.Functional;
 
-public class redis_healthcheck_should
+public class redis_healthcheck_should : IClassFixture<RedisContainerFixture>

```

Comment:
nit: we could use the primary ctors here as well?

```suggestion
public class redis_healthcheck_should(RedisContainerFixture redisContainerFixture) : IClassFixture<RedisContainerFixture>

```